data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+	source "Darkcloak"
+	to offer
+		or
+			has "<uhai> 2: Prove Your Strength: deferred"
+			has "Wanderers: Defend Vara Ke'sok: aborted"
+			random < "Unfettered Jump Drive 1: offered" * 20 + "Unfettered Jump Drive 2: offered" * 25 + "Unfettered Jump Drive 3: offered" * 30 + "Unfettered Jump Drive 4: failed" * 25
+		# 3 is what a default built Sea Scorpion has
+		"armament deterrence" >= 3
+		not "<uhai> 2: Prove Your Strength: done"
+	on offer
+		conversation
+			branch waiting
+				or
+					has "<uhai> 2: Prove Your Strength: deferred"
+					has "<uhai> 2: Prove Your Strength: offered"
+			`You start wandering in the spaceport, and on the way you notice just how much everything around you here is simple, efficient, and ordered, unlike the world of Firelode.`
+				to display
+					has "visited planet: Firelode"
+			`You start wandering in the spaceport, and on the way you are surprised how it is simple, efficient, and ordered, especially for an Unfettered planet.`
+				to display
+					not "visited planet: Firelode"
+			`	The Unfettered present are proudly showing their weapons, more so than on other planets, and some of them are even armored. They seem ready for action, at any time. Thankfully they no longer consider you an enemy, at least for the moment.`
+			`	Eventually your attention is drawn to a gathering, where someone is speaking loudly to a small crowd of people. They all seem very familiar with each other, and as you watch, they have noticed you too. The one who was addressing the others saunters through the group and over to you. This could mean trouble.`
+			`	"Arrr, be this the bilge-sucking<, unfettered title> ye all speak so highly of? The one who fears not to come into our ports, and fight our ships?" He asks the others, apparently mocking you. You're a little surprised at his creative grasp of your language.`
+			choice
+				`	"I fear nothing. Do you?"`
+					goto provoke
+				`	(Say nothing.)`
+				`	(Run away.)`
+					goto run
+			`	"See? The lily-livered monkey dares not even speak! This scallywag be no more than shark bait! Even our 'umble clipper would be sending ye straight to Davy Jones' locker! We be ready to prove it, too, yet we dare not, lest we lose our dancing monkey to the briny deep! But mateys, there be no shame if it dance away, with a dinghy looking like that!" He seems very full of himself. Perhaps someone should teach him a lesson.`
+			choice
+				`	"You dare insult the <ship>?"`
+				`	"I have no need to prove myself to a rodent."`
+					goto provoke
+				`	(Go. He's not worth it.)`
+					goto run
+			`	"Sink me! That flotsam really be your ship? Given the hornswoggle of your reputation, I pegged you for a fancier type. Your prowess is no lie, indeed then, if you land in such a scow, but can ye truly fight in it," he asks in a disrespectful tone, before turning to address the crowd once again. "I wonder if it can even launch again. Perhaps we should scuttle it right here."`
```

Comment:
```suggestion
			`	"Sink me! That flotsam really be your ship? Given the hornswoggle of your reputation, I pegged you for a fancier type. Your prowess is no lie, indeed then, if you land in such a scow, but can ye truly fight in it?" he asks in a disrespectful tone, before turning to address the crowd once again. "I wonder if it can even launch again. Perhaps we should scuttle it right here."`
```